code/modules/surgery/advanced/lobotomy.dm

```diff
@@ -52,6 +52,8 @@
 			target.gain_trauma_type(BRAIN_TRAUMA_SEVERE, TRAUMA_RESILIENCE_MAGIC)
 		if(3)
 			target.gain_trauma_type(BRAIN_TRAUMA_SPECIAL, TRAUMA_RESILIENCE_MAGIC)
+		else
+			return TRUE
 	return TRUE
 
```

Comment:
Do we need the extra return below now? As it always does If something.

Reply:
if it hits the other switch cases it will return null then

Reply:
byond has no proper syntax for "do nothing"